File: C/Factors.c
#include<stdio.h>
void main()
{
	int n,i;
	printf("Enter a number : ");
	scanf("%d",&n);
	printf("Factors are : ");
	for(i=1;i<=n;i++)
	{
		if(n%i==0)
		{
			if(n!=i)
			printf("%d, ",i);
			else
			printf("%d",i);
		}
	}
	return;
}

File: Python/Industrial Training/factors.py
n = int(input("Enter a number : "))
print("Factors :")
if n > 0:
    for i in range(1, n + 1):
        if(n % i == 0):
            print(i)
elif n < 0:
    for i in range(-1, n - 1, -1):
        if(n % i == 0):
            print(i)
else:
    print(0)